scania-digital-design-system/tegel
# feat(icons): add new icons

## **Describe pull-request**  
Added new icons: card, dashboard, dollar, face_dissatisfied, face_neutral, face_satisfied, insights and wallet. Pin icon has also been updated to align with design.

## **Issue Linking:**  
[CDEP-3062](https://tegel.atlassian.net/browse/CDEP-3062)    

## **How to test**  
1. Go to preview link below
2. Check in Foundations -> Icons
3. Make sure the newly added icons are available

## **Checklist before submission**

## **Suggested test steps**

[CDEP-3062]: https://tegel.atlassian.net/browse/CDEP-3062?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
